src/Common/src/Interop/Windows/mincore/Interop.ImpersonateNamedPipeClient.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class mincore
+    {
+#pragma warning disable BCL0015
```

Comment:
This is because the API isn't available in our approved mincore list?  Will it be?